module/userProjects/helper.js

```diff
@@ -1113,7 +1114,28 @@ module.exports = class UserProjectsHelper {
                         solutionDetails = solutionDetails.data[0];
                         
                     }
-
+                    // program join API call 
+                    let programJoined = await programUsers.findProgramJoined(solutionDetails.programId,userId)
```

Comment:
@ankitpws once you generalise the function you might have to change the parameters passed, eg: (query,projection,skipFields)